src/android/CodePushPackageManager.java

```diff
@@ -95,4 +95,16 @@ public InstallOptions getPendingInstall() {
     public void clearPendingInstall() {
         this.codePushPreferences.clearPendingInstall();
     }
+
+    public void markUnconfirmedInstall(boolean unconfirmed) {
```

Comment:
suggestion: maybe it would be easier to understand if there were two methods, markUnconfirmedInstall, and markConfirmedInstall / clearUnconfirmedInstall, instead of passing the boolean flag to determine what this method does.